File: modul1/soal8/PRAK108-2610817310007-Rahmat Setiadi.py
import math

lap = 5
distance = 14
circumference = distance / lap
radius = circumference / (2 * math.pi)

print("Diketahui :")
print("Pak Dengklek mengelilingi taman =", lap, "Putaran")
print("Jarak tempuh Pak Dengklek =", distance, "Kilometer")
print("\nJawaban :")
print(f"Jari-jari taman yang dikelilingi Pak Dengklek adalah {radius:.2f} Kilometer")

File: modul1/soal8/PRAK108-2610817310007-Rahmat Setiadi.c

#include <stdio.h>
#include <math.h>

int main()
{
    int lap = 5;
    float distance = 14;
    float circumference = distance / lap;
    float radius = circumference / (2 * M_PI);

    printf("Diketahui :\n");
    printf("Pak Dengklek mengelilingi taman = %d Putaran\n", lap);
    printf("Jarak tempuh Pak Dengklek = %.0f Kilometer\n", distance);
    printf("\nJawaban :\n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %.2f Kilometer\n", radius);

    return 0;
}
